DisCatSharp.ApplicationCommands/Workers/ApplicationCommandWorker.cs

```diff
@@ -75,9 +75,9 @@ > ParseContextMenuCommands(Type type, IEnumerable<MethodInfo> methods, List<Comm
 
 			var parameters = contextMethod.GetParameters();
 			if (parameters.Length == 0 || parameters == null || !ReferenceEquals(parameters.FirstOrDefault()?.ParameterType, typeof(ContextMenuContext)))
-				throw new ArgumentException($"The first argument must be a ContextMenuContext!");
+				throw new ArgumentException($"The first argument of the command '{contextAttribute.Name}' has to be an ContextMenuContext!");
```

Comment:
shouldn't it be ".. a ContextMenuContext"?

Reply:
F

Reply:
well next time

Reply:
Oopsies